canvas_modules/common-canvas/src/command-actions/setNodeLabelAction.js

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2017-2020 Elyra Authors
```

Comment:
Its 2021 now :)

Reply:
I fixed all the copyrights for files I've changed and created.